raiden/tests/unit/transfer/test_source_routing.py

```diff
@@ -152,6 +153,69 @@ def test_resolve_routes(netting_channel_state, chain_state, token_network_state)
     assert route_states[0].forward_channel_id == channel_id, msg
 
 
+def test_initiator_accounts_for_fees_when_selecting_routes():
+    def make_mediated_transfer_state_change(
+        transfer_amount, allocated_fee_amount, channel_capacity
```

Comment:
Would be nice to have type annotations, including the return type.

Reply:
Done